File: backend/BudgetManagementAPI/Models/Budget/BudgetDto.cs
using BudgetManagementAPI.Enums;

namespace BudgetManagementAPI.Models.Budget;

public class BudgetDto
{
    public Guid Id {  get; set; }
    public Category Category { get; set; }  
    public float Percent {  get; set; }
    public float Value {  get; set; }
    public float TotalPercentageSpent {  get; set; }
    public float TotalValueSpent {  get; set; }
    //public float RestSum {  get; set; }
}

File: budget-management-app/src/app/models/budgetDto.ts
import { Category } from '../enums/category';

export class BudgetDto {
  Id!: string;
  Category!: Category;
  Percent!: number;
  Value!: number;
  TotalPercentageSpent!: number;
  TotalValueSpent!: number;
  RestSum!: number;
}
